contents/handbook/company/working-with-design.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Working with Design
+sidebar: Handbook
+showTitle: true
+---
+
+Design is currently a shared resource at PostHog. This explains what we do, our design process, and how we can assist across the PostHog team.
+
+## Design's role at PostHog
+
+1. Support Small Teams (and contributors) in building better versions of PostHog
+1. Enable customers to build better products (using PostHog)
+1. Communicate to prospective customers the value we provide
+
+### Tangibly, we:
+
+1. Initiate new projects to support the roles listed above
+1. Support Small Teams in completing their sprint tasks
+1. Iterate based on feedback from customers
+
+## Our process
+
+Design tasks are managed with our [GitHub Org project](https://github.com/orgs/PostHog/projects/3). This aggregates design-related tasks from the main three repositories for the company:
+
+1. [PostHog app](https://github.com/PostHog/posthog) - open source repo
+1. [posthog.com](https://github.com/PostHog/posthog.com) - website + docs
+1. Internal - higher-level company strategy
+
+### How our Design board works
+
+Cards generally move from left to right.
+
+1. **Design requests** - anyone can post here. (Please follow our Design Request Process below.)
+1. **Backlog** - things on our radar
+1. **This week** - equivalent of our sprint
+1. **In progress** - tasks we've started but haven't completed
+1. **Awaiting implementation** - in development or in review
+1. **Done** - shipped!
+
+## Design Request Process
+
+Since design is currently a shared resource, the best way small design requests can be handled is by posting to the **Design request** column on our [GitHub Org project](https://github.com/orgs/PostHog/projects/3).
+
+This is generally designed for small tasks. The best way to provide detail is to create a card, then convert it to an Issue where more detail is provided.
+
+(If the project is larger in scope, please don't post to the board. Just create an Issue directly in the relevant repo and we'll move to the board when the project is better-defined.)
+
+The following format will help us triage incoming requests:
+
+1. What do you need designed and why?
+1. What is the deadline?
+
+**Note:** We may defer some design requests if we're planning a larger overhaul in the near term. For example, if a request is to create an icon, we may suggest an alternate solution (like pulling an icon from The Noun Project) if we have a larger plan for revamping all icons in a section in the near future.
+
```

Comment:
Perhaps add something around _when_ you prefer people to ask for help and how.

ie I'm working on new feature X. Should I involve design when it's a concept, should i show up with a mockup? Should I just show them what I built then ask for some iterative improvement? A few thoughts here would be interesting